build/helper/metadata_find.py

```diff
@@ -5,11 +5,17 @@ def find_parameter(name, parameters):
     return parameter[0]
 
 
-def find_size_parameter(parameter, parameters):
+def find_size_parameter(parameter_list, parameters):
     '''Returns the parameter that is used to specify the size of another parameter. Applies to 'ivi-dance' and 'passed-in'.'''
```

Comment:
reword to plural, explain that parameter_list are the parameters for which we are going to find the parameter that is used to specify their size.